examples/browser/test/workspace/search-in-workspace.ts

```diff
@@ -0,0 +1,128 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { TopPanel } from '../top-panel/top-panel';
+import { LeftPanel } from '../left-panel/left-panel';
+
+export class SearchInWorkspace {
+
+    /** accessors */
```

Comment:
the following is the definition of "accessor" from google:

```
assessor
/əˈsesər/
noun
noun: assessor; plural noun: assessors
a person who evaluates the quality of a person or thing.
"the assessor interviews the candidates and observes them actually using the language"
a person who calculates or estimates the price or value of something.
"loss assessors were beginning to get an idea of the extent of the damage"
a person who is knowledgeable in a particular field and is called upon for advice, typically by a judge or committee of inquiry.
```
i guess you meant "selectors" ?

Reply:
sorry yes, thank you!